### PR TITLE
Development

### PR DESCRIPTION
**Pull request by tomashhurst**
Major updates to multiple functions.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,10 +1,11 @@
 Package: Spectre
 Type: Package
 Title: A Toolbox for the Analysis of High-Dimensional Cytometry and Single-Cell Data
-Version: 0.3.7
+Version: 0.3.7-0
```
Comment (ghar1821) [inline DESCRIPTION:4]: Why not 0.3.8? We have a fair bit of changes here.

File: R/create.stats.R [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
-                     use.cols,
-                     sample.col,
-                     grp.col,
-
-                     comparisons, # make a tool to create a factorial comparison design -- for now just specify manually
-
-                     variance.test = "kruskal.test", ## add ANOVA
-                     pairwise.test = "wilcox.text", ## Add t-test
-                     corrections = "fdr"){
+create.stats <- function(dat,
```
Comment (ghar1821) [inline R/create.stats.R:7]: compute.stats maybe? create sounds a bit weird to me.

Comment (ghar1821) [reply R/create.stats.R:7]: We also need to write the documentation for this.

**Comment by tomashhurst** [comment]
OK I'll kill this request so I can make multiple updates, then will put up a new one when ready.

**Pull request closed without merging**